src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -317,8 +317,7 @@ public function load(array $configs, ContainerBuilder $container)
             $this->initFields($config['fields_defaults'], $container);
         }
 
-        $this->initListBuilder($container, $loader);
-
+        $loader->load('list_builder.xml');
```

Comment:
@niklasnatter all projects from 2.2 has no AdminBundle register so to make this not crash that projects upgrading from that version we should check if `kernel.bundles` parameter including SuluAdminBundle and then include the `list_builder.xml` only so we are sure the admin bundle parameters are set which are used by the `list_builder.xml`.

Example: https://github.com/sulu/sulu/blob/625b7c595392e109613631c6605f871e1b184918/src/Sulu/Bundle/WebsiteBundle/DependencyInjection/SuluWebsiteExtension.php#L138-L140